roles/jwflory.teleirc/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # defaults file for teleirc
-default_irc_server: chat.freenode.net
+default_irc_server: irc.libera.chat
 default_irc_nickserv_service: NickServ
-default_version: v1.2.2
+default_version: v2.1.0
```

Comment:
This will be changed once we cut the next TeleIRC release (`v2.1.0` does not support imgur yet).